README.md

```diff
@@ -3,19 +3,238 @@
 The Open Policy Agent (OPA) is an open source, general-purpose policy engine
 that enables unified, context-aware policy enforcement across the entire stack.
 
-## Want to learn about OPA?
+## Want to learn more about OPA?
 
 - See [openpolicyagent.org](http://www.openpolicyagent.org) to get started.
 - See [blog.openpolicyagent.org](https://blog.openpolicyagent.org) for blog posts about OPA and policy.
 - Join the conversation on [Slack](http://slack.openpolicyagent.org).
 
 ## Want to get OPA?
 
-See [GitHub releases](https://github.com/open-policy-agent/opa/releases) for binary releases.
+- See [Docker Hub](https://hub.docker.com/r/openpolicyagent/opa/tags/) for Docker images.
+- See [GitHub releases](https://github.com/open-policy-agent/opa/releases) for binary releases and changelogs.
 
-## Want to build OPA?
+## Want to integrate OPA?
 
-See [DEVELOPMENT.md](./docs/devel/DEVELOPMENT.md) for development environment setup and build instructions.
+* See
+  [![GoDoc](https://godoc.org/github.com/open-policy-agent/opa?status.svg)](https://godoc.org/github.com/open-policy-agent/opa/rego)
+  to integrate OPA into services written in Go.
+
```

Comment:
Can we add an integration option for the HTTP API if you run it as a daemon?  Maybe link to the API docs.  Otherwise, it looks like the only way to use it is as a Go Library.